### PR TITLE
Adding category and region counters to search view.

### PR DESCRIPTION
**Pull request by cynthialee**

### CODE REVIEW

File: src/angular/partials/search.html [hunk 1]
```diff
@@ -7,26 +8,30 @@
...
+				<h4>Region</h4>
+				<!-- populating the catgories from the SearchCtrl  -->
+				<div ng-repeat="region in regions">
+			        <a href="#/results?region={{region}}" class="list-group-item">
+			          <span class="icon glyphicon glyphicon-star-empty"></span>
+			          {{region.name}}
+			          <span class="badge">{{region.count}}</span>
+			        </a>
+				</div>
+             </div>
```
Comment (lkysow) [inline src/angular/partials/search.html]: Can you fix the indentation here so it stays consistent. Let's use 2 spaces for HTML

File: src/angular/app.js [hunk 2]
```diff
@@ -117,34 +117,37 @@ controllers.controller('SearchCtrl', ['$scope', '$http', '$location', 'ServicesL
...
-                regions[region] = true;
+                if (regions[region] == null) {
+                    regions[region] = 0;
+                }
+                regions[region]++;
             }
         });
 
+        // now to get an array of categories we just map over the keys of the object
+        $scope.categories = $.map(categories, function (value, index) {
```
Comment (lkysow) [inline src/angular/app.js:153]: I think you don't need the map anymore.
Do

```
$scope.categories = categories;
$scope.regions = regions;
```

And then in the view use

```
<div ng-repeat="(categoryName, count) in categories"> ... </div>
```

**Comment by lkysow** [comment]
:+1: after comments

**Comment by Jareechang** [comment]
I guess as you mentioned earlier @cynthialee, we could probably put regions and categories into the same loop. Either way, looks good :+1:

File: src/angular/partials/search.html (hunk 1 above)
Comment (cynthialee) [inline src/angular/partials/search.html]: Looks like you guys are replacing your tabs with spaces?

**Comment by cynthialee** [comment]
@lkysow @Jareechang Take another look please. I've changed the spacing. I've kept the var mappings because of new data (activity vs category, i.e. FOOD includes Food Security and Nutritional Status Impovement) and to make it easier semantically.

**Comment by Jareechang** [comment]
nice! :+1:

**Pull request merged**